src/containers/BooksForm.js

```diff
@@ -1,41 +1,90 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createBook } from '../actions/index';
+import getId from '../helpers/getId';
 
-const BooksForm = () => {
-  const categories = [
-    'Action',
-    'Biography',
-    'History',
-    'Horror',
-    'Kids',
-    'Learning',
-    'Sci-Fi',
-  ];
-
-  return (
-    <form>
-      <label htmlFor="title">
-        Title
-        <input type="text" id="title" />
-      </label>
-      <label htmlFor="category">
-        Choose a category:
-        <select id="category">
+const categories = [
+  'Action',
+  'Biography',
+  'History',
+  'Horror',
+  'Kids',
+  'Learning',
+  'Sci-Fi',
+];
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+
```

Comment:
I guess you'll surely need to put the default category in here, instead of empty string